### PR TITLE
[WIP] Add type hints for Lxmert (TF)

### PR DESCRIPTION
**Pull request by elusenji**
# What does this PR do?

This PR adds type hints to the Lxmert model for TensorFlow.

Models:

Lxmert: @LysandreJik

### CODE REVIEW

File: src/transformers/models/lxmert/modeling_tf_lxmert.py [hunk 1]
```diff
@@ -951,17 +958,17 @@ def __init__(self, config, *inputs, **kwargs):
...
+        visual_pos: Optional[tf.Tensor] = None,
+        attention_mask: Optional[Union[np.ndarray, tf.Tensor]] = None,
+        visual_attention_mask: Optional[Union[np.ndarray, tf.Tensor]] = None,
+        token_type_ids: Optional[Union[np.ndarray, tf.Tensor]] = None,
+        inputs_embeds: Optional[Union[np.ndarray, tf.Tensor]] = None,
+        output_attentions: Optional[bool] = None,
+        output_hidden_states: Optional[bool] = None,
+        return_dict: Optional[bool] = None,
+        training: bool = False,
     ):
```
Comment (Rocketknight1) [inline src/transformers/models/lxmert/modeling_tf_lxmert.py:972]:
```suggestion
    ) -> Union[Tuple, TFLxmertModelOutput]:
```

**Review by Rocketknight1 — APPROVED** [review]
Looks perfect to me, thanks! I threw in the output type too, but all of your argument annotations were perfect.

**Comment by elusenji** [comment]
@Rocketknight1 Is there anything else I need to add to this PR?

**Comment by Rocketknight1** [comment]
@elusenji No, sorry! I meant to merge it after the tests passed but lost track of it yesterday. Doing it now, and thanks again!

**Pull request merged**